src/js/components/Table/Paginated/DateFilter.tsx

```diff
@@ -65,6 +68,12 @@ const formatDate = (x?: string) => {
   return date;
 };
 
+const formatDisplayDate = (date: string, locale?: string | string[]): string => {
+  return new Intl.DateTimeFormat(
+    locale ? locale : navigator.languages ? [...navigator.languages] : navigator.language
+  ).format(formatDate(date));
+};
```

Comment:
Thoughts on just making locale a ` locale?: string[]`? If we do that, we may want to name that `locales` instead of `locale`.